ynput/OpenPype
# AYON workfiles tools: Revisit workfiles tool

## Changelog Description
Revisited workfiles tool for AYON mode to reuse common models and widgets.

## Additional info
Reused models and widgets from `ayon_utils`. Button "Go to current context" works. Refresh button refreshes workfiles too.

## Testing notes:
1. Launch DCC/host of your choice
2. Open workfiles tool
3. Try few standard things with workfiles tool
- Try to open/save workfiles
- Change context and open/save
- Save artist note
- Open published file
- Copy published file
- Hit refresh (should refresh all folders, tasks and workfiles)
